themes/default/content/blog/benchmarking-python-performance/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Benchmarking Python Performance"
+date: 2023-07-31
+meta_desc: "Benchmarking and improving the performance of Pulumi Python programs."
+meta_image: meta.png
+
+authors:
+    - robbie-mckinstry
+
+tags:
+    - performance
+    - platform
+    - engineering
+---
+
+This is the second post in a series about performance optimizations we've made
+to the Pulumi CLI and SDKs. In this post, we'll go deep on a performance
+improvement we made for Pulumi Python programs. You can read more
+about Amazing Performance in
+[the first post in the series](https://www.pulumi.com/blog/amazing-performance/).
+
+<!--more-->
+
+We recently took a hard look at the performance of Python programs when we
```

Comment:
It might be worthwhile to reconsider the use of "recently" and similar words, given that this fix was implemented a fair amount of time ago